assets/js/blocks/cart-checkout/checkout/order-summary-item.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * External dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+import { getCurrency } from '@woocommerce/base-utils';
+import { useStoreCartItemQuantity } from '@woocommerce/base-hooks';
+import Label from '@woocommerce/base-components/label';
+import {
+	ProductImage,
+	ProductLowStockBadge,
+	ProductMetadata,
+	ProductName,
+	ProductPrice,
+} from '@woocommerce/base-components/cart-checkout';
+import PropTypes from 'prop-types';
+
+const CheckoutOrderSummaryItem = ( { cartItem } ) => {
+	const {
+		images,
+		low_stock_remaining: lowStockRemaining = null,
+		name,
+		permalink,
+		prices,
+		summary,
+		variation,
+	} = cartItem;
+
+	const { quantity } = useStoreCartItemQuantity( cartItem );
```

Comment:
I don't think you need this hook - you can just accept a static quantity prop from parent component (which should have all the item details).

`useStoreCartItemQuantity` hook is for changing quantity or removing item :)

Reply:
Oh, good catch. I missed that `cartItem` has the `quantity` attribute. Fixed in 87214e31c0bb18ccdcd9e22ab76ab278f95e48eb.